src/screens/ProfileScreen/index.js

```diff
@@ -608,7 +613,7 @@ const ProfileScreen = ({route}) => {
     return reloadFetchAnonymousPost();
   };
   return (
-    <View style={styles.container} forceInset={{top: 'always'}}>
+    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The root component has been changed from `View` to `SafeAreaProvider`. Make sure this doesn't affect the layout or functionality of the component.
```diff
- <View style={styles.container} forceInset={{top: 'always'}}>
+ <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->